File: fagsystem/src/main/kotlin/no/david/fagsystem/model/Avtale.kt
package no.david.fagsystem.model

import javax.persistence.*

@Entity
class Avtale(
    var registreringsnummer: String,
    var bonus: Int,
    var status: String,
    @OneToOne
    @JoinColumn(name = "kunde_id")
    var kunde: Kunde,
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY) var id: Long? = null
)

File: frontend/src/model/Avtale.ts
export type Avtale = {
  id: number;
  registreringsnummer: string;
  bonus: string;
  kunde: Kunde;
};

export type Kunde = {
  id: number;
  fodselsnummer: string;
  fornavn: string;
  etternavn: string;
  epost: string;
};
